File: Anyelo Carrillo C++/Condicionales If Else.cpp
#include <iostream>
#include <string>

int main() {
    int a = 2;

    if (a == 2) {
        std::cout << "a vale 2" << std::endl;
    } else {
        std::cout << "a es diferente de 2" << std::endl;
    }

    std::string Nombre = "Esteban";
    int Edad = 28;
    std::string Pais = "Colombia";

    if (Nombre == "Esteban" && Edad == 28 && Pais == "Colombia") {
        std::cout << "Su nombre es " << Nombre << ", tiene " << Edad << " años y es de " << Pais << std::endl;
    } else if (Nombre == "Esteban" && !(Edad == 28)) {
        std::cout << "Su nombre es Esteban y no tiene 28 años" << std::endl;
    } else if (Nombre != "Esteban" && Edad == 28) {
        std::cout << "Su nombre no es Esteban y tiene 28 años" << std::endl;
    } else {
        std::cout << "No se llama Esteban ni tiene 28 años" << std::endl;
    }

    return 0;
}
         

File: Anyelo Carrillo Java/Condicionales If Else.java
public class Main {
    public static void main(String[] args) {
        int a = 2;
        if (a == 2) {
            System.out.println("a vale 2");
        } else {
            System.out.println("a es diferente de 2");
        }

        String Nombre = "Esteban";
        int Edad = 28;
        String Pais = "Colombia";

        if (Nombre.equals("Esteban") && Edad == 28 && Pais.equals("Colombia")) {
            System.out.println("Su nombre es " + Nombre + ", tiene " + Edad + " años y es de " + Pais);
        } else if (Nombre.equals("Esteban") && Edad != 28) {
            System.out.println("Su nombre es Esteban y no tiene 28 años");
        } else if (!Nombre.equals("Esteban") && Edad == 28) {
            System.out.println("Su nombre no es Esteban y tiene 28 años");
        } else {
            System.out.println("No se llama Esteban ni tiene 28 años");
        }
    }
}

         